packages/server/src/tests/testsStore.ts

```diff
@@ -0,0 +1,36 @@
+import * as AWS from 'aws-sdk';
+import { isProd } from '../lib/env';
+import { logError } from '../utils/logging';
+
+const stage = isProd ? 'PROD' : 'CODE';
+
+export type ChannelTypes =
+    | 'Epic'
+    | 'EpicAMP'
+    | 'EpicAppleNews'
+    | 'EpicLiveblog'
+    | 'EpicHoldback'
+    | 'Banner1'
+    | 'Banner2'
+    | 'Header';
+
+export const getTests = <T>(channel: ChannelTypes): Promise<T[]> =>
+    queryChannel(channel, stage)
+        .then(tests => (tests.Items ?? []).sort((a, b) => a.priority - b.priority) as T[])
+        .catch(error => {
+            logError(`Error reading tests from Dynamo: ${error.message}`);
```

Comment:
let's also send a cloudwatch metric so that this triggers an alarm - https://github.com/guardian/support-dotcom-components/blob/main/packages/server/src/utils/cloudwatch.ts#L14
There's already a `channel-tests-error` value

Reply:
Added!